packages/react/src/vertex3.tsx

```diff
@@ -0,0 +1,205 @@
+import { Utility } from "@hpcc-js/common";
+import * as React from "@hpcc-js/preact-shim";
+import { Icon } from "./icon";
+import { TextBox } from "./text";
+import { Vertex } from "./vertex";
+
+export const Annotation = ({
+    text = "",
+    padding = 2,
+    gutter = 4,
+    cornerRadius,
+    shape = "rectangle",
+    height = 16,
+    width = 16,
+    textColor = "#FFFFFF",
+    backgroundColor = "#ED1C24",
+    shapeOptions = {
+        shape: "rectangle",
+        fill: "#ED1C24",
+        stroke: "#555555",
+        strokeWidth: 1
+    },
+    textOptions = {
+        fontFamily: "Verdana",
+        fontSize: 12,
+        fill: "#555555"
+    }
+}) => {
+    return <>
+        <TextBox
+
+            text={text}
+            fontFamily={textOptions.fontFamily}
+            height={textOptions.fontSize}
+            textFill={textOptions.fill}
+
+            fill={shapeOptions.fill}
+            stroke={shapeOptions.stroke}
+            strokeWidth={shapeOptions.strokeWidth}
+
+            padding={padding}
+            cornerRadius={cornerRadius}
+        ></TextBox>
+    </>;
+};
```

Comment:
As discussed Annotation + Icon should probable be the same thing?

Then this becomes a variant of TextBox - IconTextBox or similar?